code/modules/hydroponics/beekeeping/smoker.dm

```diff
@@ -23,7 +23,7 @@
 
 //I would prefer to rename this to attack(), but that would involve touching hundreds of files.
 /obj/item/weapon/bee_smoker/resolve_attackby(atom/A, mob/user)
-	if (istype(A, /obj/structure/reagent_dispensers/fueltank) && get_dist(src,A) <= 1)
```

Comment:
If it is using woodpulp now, the sink interaction should be scrapped or changed to something else?

Reply:
Most definitely. almost forgot lmao. Need to think of a fix for this, maybe commercial woodpulp canisters the shop can sell.

Reply:
Fixed.